packages/jsdoc-plugins/lib/relation-fixer/addmissingdoclets.js

```diff
@@ -67,11 +85,11 @@ function addMissingDoclets( originalDoclets ) {
 		}
 	];
 
-	for ( const childDoclet of entitiesWhichNeedNewDoclets ) {
+	for ( const extensibleDoclet of extensibleDoclets ) {
```

Comment:
`s/extensibleDoclet/doclet`?

Reply:
It's more precise.

Reply:
And it's really hard to see difference between `extensibleDoclet` and `extensibleDoclets`.